build-system/tasks/e2e/functional-test-controller.js

```diff
@@ -44,6 +44,21 @@ class ElementHandle {
   }
 }
 
+/**
+ * Keys to send to the FunctionalTestController#type method to trigger
+ * actions instead of text.
+ * @enum {string}
+ */
+export const Keys = {
```

Comment:
We import `Key` from selenium-webdriver in `selenium-webdriver-controller.js`. Can that be used instead?

Reply:
This is part of the abstraction layer so we can use the same keys when using Selenium as our webdriver and when using Puppeteer without rewriting the tests. The `Key` from `selenium-webdriver` wouldn't work for Puppeteer.

This mapping is also good because it lets us use the browser-standard names from https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key and selenium uses non-standard values

https://github.com/SeleniumHQ/selenium/blob/df40c28b41d4b3953f90eaff84838a9ac052b84a/javascript/node/selenium-webdriver/lib/input.js#L44-L57

Reply:
Okay I see!